.github/actions/spelling/expect.txt

```diff
@@ -3,6 +3,7 @@ adfdb
 admissionregistration
 alertmanager
 amazonaws
+amazonvpc
```

Comment:
IIUC this was just resolved in another PR.

Reply:
Sadly, that word isn't in the additional dictionary: https://github.com/check-spelling/cspell-dicts/blob/master/dictionaries/aws/aws.txt#L52

I'd take a PR to add it (and other things)...